### PR TITLE
[loki-simple-scalable] add initContainers

### PR DESCRIPTION
**Pull request by Grounz**
Hi,
I submit this pr beacause when we want deploy loki simple-scalable helm charts we use boltdb-shipper.

When read and write loki pod start they have a var/loki/boltdb-shipper-active: permission denied errir and we cannot fix that without use an initContainer which do chown 10001:10001 /var/loki.

Loki-simple-scalable helm chart not containers initContainers in statefulset templates for read and write pods.

So i have add the possibility to add initContainers for read and for write pods.

in values.yaml:

```
write:
...
  initContainers: []
read:
...
  initContainers: []
```

Signed-off-by: lenglet-k <lenglet-k@mgdis.fr>

### CODE REVIEW

**Comment by zanhsieh** [comment]
@Grounz 
Please bump the Chart.yaml to latest and sign license/cla. Thank you.

File: charts/loki-simple-scalable/templates/read/statefulset-read.yaml [hunk 1]
```diff
@@ -38,6 +38,8 @@ spec:
       {{- include "loki.readPriorityClassName" . | nindent 6 }}
       securityContext:
         {{- toYaml .Values.loki.podSecurityContext | nindent 8 }}
+      initContainers:
```
Comment (trevorwhitney) [inline charts/loki-simple-scalable/templates/read/statefulset-read.yaml:41]: WDYT of wrapping this in a `with` block so the key is only included when init containers are provided?

**Review by trevorwhitney — CHANGES_REQUESTED** [review]
Would prefer a `with` block so key is excluded rather than included with empty array. Also, please sign CLA.

**Comment by zanhsieh** [comment]
@trevorwhitney @joe-elliott 
Documentation CI seems having some problem.

**Comment by zanhsieh** [comment]
@Grounz 
Since loki and loki-simple-scalable moved to https://github.com/grafana/loki/tree/main/production/helm/loki . Please PR over there.

**Pull request closed without merging**